module/VuFind/src/VuFind/Db/Entity/PaymentFee.php

```diff
@@ -0,0 +1,370 @@
+<?php
+
+/**
+ * Entity model for payment_fee table
+ *
+ * PHP version 8
+ *
+ * Copyright (C) The National Library of Finland 2025.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Database
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     http://vufind.org   Main Site
+ */
+
+declare(strict_types=1);
+
+namespace VuFind\Db\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Entity model for payment_fee table
+ *
+ * @category VuFind
+ * @package  Database
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     http://vufind.org   Main Site
+ */
+#[ORM\Table(name: 'payment_fee')]
+#[ORM\Index(name: 'payment_fee_payment_id_idx', columns: ['payment_id'])]
+#[ORM\Entity]
+class PaymentFee implements PaymentFeeEntityInterface
+{
+    /**
+     * Unique ID.
+     *
+     * @var int
+     */
+    #[ORM\Column(name: 'id', type: 'integer', nullable: false)]
+    #[ORM\Id]
+    #[ORM\GeneratedValue(strategy: 'IDENTITY')]
+    protected $id;
+
+    /**
+     * Payment.
+     *
+     * @var Payment
+     */
+    #[ORM\JoinColumn(name: 'payment_id', referencedColumnName: 'id', nullable: false, onDelete: 'CASCADE')]
+    #[ORM\ManyToOne(targetEntity: \VuFind\Db\Entity\Payment::class)]
+    protected $payment;
+
+    /**
+     * Title
```

Comment:
This is the only property missing a period for some reason.

```suggestion
     * Title.
```